addons/website_slides_survey/static/src/js/slides_certification_upload_toast.js

```diff
@@ -0,0 +1,34 @@
+odoo.define('website_slides.certification_upload_toast', function (require) {
+'use strict';
+
+var publicWidget = require('web.public.widget');
+
+var sessionStorage = window.sessionStorage;
+var core = require('web.core');
+var _t = core._t;
+
+
+publicWidget.registry.CertificationUploadToast = publicWidget.Widget.extend({
```

Comment:
Please return the widget so that it can be properly extended.

Reply:
Yessir